@types/lessonCard.d.ts

```diff
@@ -14,6 +14,7 @@ export type ReviewButtonProps = {
   isCompleted: boolean
   reviewUrl: string
   lessonId: number
+  style?: string
```

Comment:
`style` should be renamed to `additionalClass` or something. `style` is a misnomer.

Reply:
I renamed it to className.